packages/@react-types/table/src/index.d.ts

```diff
@@ -151,3 +153,54 @@ export interface TableCollection<T> extends GridCollection<T> {
   /** The node that makes up the body of the table. */
   body: GridNode<T>
 }
+
+export interface ITableColumnLayout<T> {
```

Comment:
Does this need to be in react-types as an interface? Could we just use the real class from stately as the type? I don't think we'll ever have multiple implementations of this.

Reply:
I was having some trouble with a circular dependency, layout -> table -> layout
Looking at it again, looks like it was from something else I resolved, not this. Is this the direction we want things to go though? Where the layout package depends on table? seems like it should be the other way around?